src/extensions/react-flow/_config.scss

```diff
@@ -15,3 +15,4 @@ $reactflow-edge-stroke-color-selected: $eccgui-color-accent !default;
 $reactflow-transition-time: 0.25s !default;
 $reactflow-transition-function: "" !default;
 $reactflow-transition-anglestart: -90deg;
+$react-flow-cursor-delimiter-offset: -6.5 * $reactflow-node-border-width;
```

Comment:
Please keep the naming pattern like the other `$reactflow-*` variables (not `$react-flow-*`).
And this white space value should not be bound to a border width. It should be probably `$eccgui-size-block-whitespace` (or half of it).